examples/documentation/CMakeLists.txt

```diff
@@ -32,3 +33,8 @@ foreach(example_program ${example_programs})
     pika_add_example_test("documentation" ${example_program} ${${example_program}_PARAMETERS})
   endif()
 endforeach()
+
+set_tests_properties(
+  tests.examples.documentation.require_started_documentation PROPERTIES PASS_REGULAR_EXPRESSION
+                                                                        "terminate called"
+)
```

Comment:
I think this needs to be wrapped in `if (PIKA_WITH_TESTS AND PIKA_WITH_TESTS_EXAMPLES)` (https://app.circleci.com/pipelines/github/pika-org/pika/11893/workflows/05afdcaf-c1be-4eed-bca5-48369a22b124/jobs/131879).